src/Connection/PostObjects.php

```diff
@@ -515,7 +515,9 @@ public static function get_connection_args( $args = [], $post_type_object = null
 				'description' => __( 'Filter the connection based on dates', 'wp-graphql' ),
 			],
 			'mimeType'    => [
-				'type'        => 'MimeTypeEnum',
+				'type'        => [
```

Comment:
If we make this change, it will be a breaking change to the Schema. 

We could introduce a new `mimeTypes` field which is a `list_of => MimeTypeEnum` and then map that new input in the connection resolver.